content/working_with/manager/using-secrets.md

```diff
@@ -8,27 +8,25 @@ aliases: /manager/using-secrets/
 ---
 
 The secrets store provides a secured variable storage (key-value pairs) for data that you do not want to expose in plain text in Cloudify blueprints, such as login credentials for a platform.
-The secrets values are being stored encrypted in our DB. We use the Fernet encryption algorithm of cryptography library,
-it's a symmetric encryption method which makes sure that the message encrypted cannot be manipulated/read without the key.
-When you create a secret, the key value can be a text string or it can be a file that contains the key value. With the secrets store, you can make sure all secrets (for example credentials to IaaS environments) are stored separately from blueprints, and adhere to isolation requirements between different tenants. You can include the secret's key in your blueprints and not include the actual values in the blueprints.
+The values of the secrets are encrypted in the database.
+We use the Fernet encryption of cryptography library, which is a symmetric encryption method that makes sure that the message encrypted cannot be manipulated/read without the key.
+When you create a secret, the key value can be a text string or it can be a file that contains the key value. The secret store lets you make sure all secrets (for example credentials to IaaS environments) are stored separately from blueprints, and that the secrets adhere to isolation requirements between different tenants. You can include the secret key in your blueprints and not include the actual values in the blueprints.
 For more information, see the [get_secret]({{< relref "developer/blueprints/spec-intrinsic-functions.md#get-secret" >}}) intrinsic function.
 
-### Hidden value secret
+### Secrets with a hidden value
 
-All the secrets values are encrypted in the DB. When you create a secret you can specify if you want its value to be hidden or not.
-A hidden-value secret means the secret's value will only be shown to the user who created it, tenant managers and sys-admins.
-Use of the secret is allowed according to the users roles and the visibility of the secret.
+All values for secrets are encrypted in the database. When you create a secret you can specify if you want its value to be hidden or not.
```

Comment:
I'm not sure that "All values for secrets" is clear that it's the secret's value